### PR TITLE
Add ELK base analytics solution for APIM 4.0.0

### PR DESCRIPTION
**Pull request by ruthryi**
A - 1774

### CODE REVIEW

File: en/docs/api-analytics/cloud-analytics/elk-installation-guide.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+
+
+#### Installing Filebeat
+
+1. [Install Filebeat](https://www.elastic.co/guide/en/beats/filebeat/7.13/filebeat-installation-configuration.html#installation) according to your operating system.
+
+2. Configure **Filebeats** to read the log file in the `repository/logs` folder. 
+
+    ```
+    filebeat.inputs:
```
Comment (AnuGayan) [inline en/docs/api-analytics/cloud-analytics/elk-installation-guide.md:98]: These configurations have to be correctly indented as yaml configurations

File: en/docs/api-analytics/cloud-analytics/elk-installation-guide.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+    # The Logstash hosts
+    hosts: ["{LOGSTASH_URL}:5044"]
+    ```
+
+#### Installing Logstash
+
+1. [Install Logstash](https://www.elastic.co/guide/en/logstash/current/installing-logstash.html) according to your operating system.
+
+    ```
+    input {
```
Comment (AnuGayan) [inline en/docs/api-analytics/cloud-analytics/elk-installation-guide.md:114]: Please indent these as well

File: en/docs/api-analytics/cloud-analytics/elk-installation-guide.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+
+5. Download the artifact file [here]({{base_path}}/assets/img/analytics/cloud/export.ndjson).
+
+6. Navigate to **Stack Management** > **Saved Object** and click on **Import**. Add the downloaded artifact file as an import object, and import
+
+!!! info
+    Follow the recommendations of Elastic in order to optimize the performance of the system.
+
+
+### Step 3 - Configure Security in ELK
```
Comment (AnuGayan) [inline en/docs/api-analytics/cloud-analytics/elk-installation-guide.md:166]: This topic can be the Step 3, and inside of this,

-  Configure Basic Authentication
-  Configure Single-Sign-On with WSO2 API Manager via OpenID Connect
has to be subtopics.

**Review by AnuGayan — CHANGES_REQUESTED** [review]

File: en/docs/api-analytics/cloud-analytics/elk-installation-guide.md (hunk 3 above)
Comment (ruthryi) [reply en/docs/api-analytics/cloud-analytics/elk-installation-guide.md:170]: Added a note mentioning that either of these 2 steps can be followed.

File: en/docs/api-analytics/cloud-analytics/elk-installation-guide.md [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+    ``` yaml
+    filebeat.inputs:
+    -   type: log
+        enabled: true
+        paths:
+            - {apim_home}/repository/logs/apim_metrics.log
+        include_lines: ['(apimMetrics):']
+    output.logstash:
+    # The Logstash hosts
+    hosts: ["{LOGSTASH_URL}:5044"]
```
Comment (AnuGayan) [inline en/docs/api-analytics/cloud-analytics/elk-installation-guide.md:106]:
```suggestion
         hosts: ["{LOGSTASH_URL}:5044"]
```

**Review by AnuGayan — APPROVED** [review]

**Pull request merged**